_posts/en/newsletters/2023-02-08-newsletter.md

```diff
@@ -0,0 +1,130 @@
+---
+title: 'Bitcoin Optech Newsletter #237'
+permalink: /en/newsletters/2023/02/08/
+name: 2023-02-08-newsletter
+slug: 2023-02-08-newsletter
+type: newsletter
+layout: newsletter
+lang: en
+---
+This week's newsletter summarizes a discussion about storing data in
+transaction witnesses and references a conversation about mitigating LN
+jamming.  Also included are our regular sections with the summary of a
+Bitcoin Core PR Review Club meeting, announcements of new releases and
+release candidates, and descriptions of notable changes to popular
+Bitcoin infrastructure software.
+
+## News
+
+- **Discussion about storing data in the block chain:** users of a new
+  project recently began storing large amounts of data in the witness
+  data for segwit v1 ([taproot][topic taproot]) transactions.   Robert
+  Dickinson [posted][dickinson ordinal] to the Bitcoin-Dev mailing list
+  to inquire about whether a size limit should be imposed to discourage
+  such data storage.
+
+    Andrew Poelstra [replied][poelstra ordinal] that there is no
+    effective way to prevent data storage.  Adding new restrictions to
+    prevent unwanted data storage in witnesses would undermine
+    advantages discussed during taproot's design (see [Newsletter
+    #65][news65 tapscript]) and would likely only result in the data
+    being stored in other ways.  Those other ways might raise costs for
+    those generating the data---but probably not by enough to
+    significantly discourage the behavior---and the alternative storage
+    methods might create new problems for traditional Bitcoin users.
```

Comment:
We're beginning to see ML proposals for storing this data offchain, like "Purely off-chain coin colouring'" https://lists.linuxfoundation.org/pipermail/bitcoin-dev/2023-February/021396.html by Anthony Towns.